packages/functionals/botpress-qna/src/index.js

```diff
@@ -125,31 +125,50 @@ module.exports = {
 
     router.post('/', async (req, res) => {
       try {
+        bp.events.emit('toast.qna-save', { text: 'QnA In Progress', type: 'info', time: 120000 })
+
         const id = await storage.saveQuestion(req.body)
         res.send(id)
+
+        bp.events.emit('toast.qna-save', { text: 'QnA Success', type: 'success' })
```

Comment:
Should it be (here and in other parts of this file) `QnA Save Success/...`?

Reply:
I mean the current wording generally looks strange to me, it's not that `QnA` itself is done, it's that _saving the new QnA data_ is in progress or done successfully